bindings/python/iota_sdk/types/node_info.py

```diff
@@ -64,6 +64,12 @@ class RentStructure:
     vByteFactorData: int
     vByteFactorKey: int
 
+    def as_dict(self):
+        """Converts this object to a dict.
```

Comment:
dont we already have this using our custom @json ? why arent they on here

Reply:
Yes that's 2.0 only :(